src/FluentAssertions.Analyzers.Tests/Tips/XunitTests.cs

```diff
@@ -592,9 +600,11 @@ public void AssertIsAssignableFrom_TestCodeFix(string oldAssertion, string newAs
         [DataRow(
             /* oldAssertion: */ "Assert.IsNotAssignableFrom(typeof(string), actual);",
             /* newAssertion: */ "actual.Should().NotBeAssignableTo<string>();")]
+#if TO_BE_FIXED
```

Comment:
if this is incorrect then let's fix the expected assertion instead of ignoring (same for other tests)

Reply:
I totally agree but I just felt it was out of scope of this PR.

Reply:
I proposed a fix.